tests/test_components/test_FreqRange.py

```diff
@@ -0,0 +1,186 @@
+"""Test class FreqRange for frequency and wavelength handling"""
```

Comment:
Make sure any code is in double backslash quotes, just for consistency. more info
https://www.notion.so/flexcompute/Python-Style-Guide-afac5dbbf949463e89c18fe39b7453c9